app/scenes/Document/components/Header.js

```diff
@@ -170,7 +304,7 @@ function DocumentHeader({
                 <TemplatesMenu document={document} />
               </Action>
             )}
-            {!isEditing && (!isMobile || !isTemplate) && (
+            {!isEditing && !isMobile && !isTemplate && can.share && (
```

Comment:
Not sure what happened here, doesn't seem like it should be edited at all

Reply:
Previous logic will show `Share` on the mobile if the doc isn't a template.

It was collapsing with the TOC icon. What I inferred is that we don't want to show the `Share` on mobile.

Reply:
Okay, just the `&& can.share` that isn't needed – we always show the share button now, even if public sharing is disabled